.github/workflows/ci.yml

```diff
@@ -14,10 +14,19 @@ jobs:
         toxenv: [py]
         include:
           # TODO: Change to 3.x once pylint fully supports Python 3.9
-          - python-version: 3.8
+          - python-version: '3.8'
             os: ubuntu-latest
             toxenv: lint
-
+          - python-version: '3.7'
+            os: windows-latest
```

Comment:
Thanks for working on this! Isn't it sufficient to add windows-latest to the os field above? We want lint outside because running it on one environment is fine.

Reply:
So for some reason, it seems that Windows builds fail on Python 3.9 but work well with all other Python versions which is why I explicitly specify the versions it works with. I discussed some possible causes in the Contribfest but this was one of the possible solutions to that. What do you think?

Reply:
Huh odd. @lukpueh what do you think?

Reply:
That's odd. Do you have an error log? Are you sure that it was because of 3.9? We do have some flakey tests. Maybe it failed on 3.9 by coincidence.

Reply:
Looks like a coincidence. I had issues on 3.8 and 3.10 just now and re-run of failed jobs cleared it up (https://github.com/adityasaky/in-toto/actions/runs/3858306634/jobs/6576722830). My suggestion is we leave this enabled alongside AppVeyor for the time being and use AV's Windows results over Github Actions when GA fails. Long term, we should study which tests fail on Windows and why they're flakey in that environment specifically.